source/SemVerAnalyzer/Assembly/MethodSignatureComparer.cs

```diff
@@ -0,0 +1,113 @@
+using System.Collections.Generic;
+using System.Linq;
+
+using dnlib.DotNet;
+
+namespace Pushpay.SemVerAnalyzer.Assembly
+{
+	public class MethodSignatureComparer : IEqualityComparer<MethodDef>
+	{
+		public static readonly MethodSignatureComparer Instance = new MethodSignatureComparer();
+
+		public bool Equals(MethodDef x, MethodDef y)
+		{
+			if (x == null && y == null) {
+				return true;
+			}
+
+			if (x == null || y == null) {
+				return false;
+			}
+
+			return EqualNames(x, y) &&
+				   EqualGenericParameters(x, y) &&
+				   EqualTypes(x.ReturnType, y.ReturnType) &&
+				   EqualParameters(x, y);
```

Comment:
looks like there's some code in here that doesn't match the checked in settings 😛